iphone/Classes/TiConsole.m

```diff
@@ -14,4 +14,37 @@ - (void)log:(NSArray *)args
   [self logMessage:args severity:@"info"];
 }
 
+- (void)time:(NSArray *)args
+{
+  if (!_times) {
+    _times = [[NSMutableDictionary alloc] init];
+  }
+  NSString *label = [args componentsJoinedByString:@""] ?: @"default";
```

Comment:
Should a developer be able to pass an array of argumenzts, e.g. `console.time(['test', 'foo'])`? If not, change the arg-type to `id` and use `ENSURE_SINGLE_ARG(args, NSString *);` to force a string directly.